script/release-postinstall.js

```diff
@@ -76,7 +74,13 @@ const copyPlatformBinaries = platformPath => {
 
 const arch = find_arch();
 
-const platformPath = "platform-" + platform + "-" + arch;
+const getPlatformFolderName = () => {
```

Comment:
I wanted to call out this method I added as not ideal. If you could help me understand how the platform folders are named, then we can adjust the platform name folder for windows to be 'win32'

Reply:
@bdunn313 So if I'm understanding your question correctly, I think the platform folders are named inside the `pipeline.yml` GH action workflow.

In particular, see the `Download windows artifacts` step under the `publish` job. It looks like the Windows artifacts (aka the folder containing the tailwind-ppx and use-tailwind-ppx binaries) is placed in the path `_release/platform-windows-x64`. What this means (I think) is that any windows `platform` (from a Node.js perspective) should cause the `getPlatformFolderName` function to returns `"platform-windows-" + arch`. 

Therefore, I think your code works!

Reply:
Ah! Makes sense. So now this code will cover both if `platform === 'windows' || platform === 'win32'` which is what we want. Thanks for explaining! I think this is fine as-is then.